packages/video_player/video_player/pigeons/messages.dart

```diff
@@ -1,5 +1,10 @@
 import 'package:pigeon/pigeon_lib.dart';
 
+class InitializeMessage {
```

Comment:
@gaaclarke might know if there's some way to tell pigeon to prefix or postfix class names to avoid clashes, or if I'm just being paranoid about this.

My concern is about this generating `FLTInitializeMessage` in the objc implementation.

Reply:
For the objc generator there is a flag that allows you to add a prefix to generated classes.  If you look a couple lines lower we are specifying FLT as the prefix in this line: `opts.objcOptions.prefix = 'FLT';`

Reply:
Ahh nice, maybe we want it to be something like `FLTVideoPlayer`.